projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/jupyter_notebook.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+from pathlib import Path
+
+
+def remove_outputs_from_notebook(notebook_path):
+    with notebook_path.open("r") as f:
+        content = json.load(f)
+
+    for cell in content["cells"]:
+        if cell["cell_type"] == "code":
+            cell["outputs"] = []
+            cell["execution_count"] = None
```

Comment:
I am not fan of this hard-coding and manual stuff. 

This will be hard to maintain. If notebooks change their structure. 
This is generally error prone. How sure are you this is enough and everything you need to clean. 

We are manipulating cells in multiple places manually. Fortunately it's rather few places and code is not big. But it's a concern. 

Have you seen  https://nbconvert.readthedocs.io/en/latest/api/preprocessors.html#removing-cells-inputs-and-outputs

Reply:
i have changed the implementation to work with it